pontoon/base/models.py

```diff
@@ -627,7 +635,30 @@ def for_project_locale(self, project, locale, paths=None):
             obsolete=False
         )
 
-        if paths:
+        translations = Translation.objects.filter(locale=locale)
+
+        # Filter by search parameters
+        if search:
+            q = search.get('q', None)
+            i = '' if search.get('casesensitive', None) else 'i'
```

Comment:
This is a bit awkward but again I have no better solution. Is case sensitivity useful anyway? Could we maybe just make all the searches case-insensitive by default?